docs/content/preview/contribute/docs/docs-build.md

```diff
@@ -37,6 +37,14 @@ To run the docs site locally and edit the docs, you'll need:
 
 * **Go**: `brew install go` installs the latest version.
 
+* **Python**: You need `python` to be available somewhere in your shell path.
+
+    Recent versions of macOS have only a `python3` executable, as does the Homebrew install. Add a symlink somewhere in your PATH. To add a symlink to the system python, you can run a command such as the following:
```

Comment:
Do we want to also mention the homebrew python3 (3.10.9 right now), or leave well enough alone with the system one (3.9.6 on macOS 13.1)?

Reply:
Homebrew is installing 3.10.9 right now so it makes sense to have latest?